docs/ref/paginator.txt

```diff
@@ -56,6 +56,31 @@ For examples, see the :doc:`Pagination topic guide </topics/pagination>`.
     ``False`` and ``object_list`` is  empty, then an ``EmptyPage`` error will
     be raised.
 
+.. attribute:: Paginator.error_messages
```

Comment:
Almost the same documentation as https://docs.djangoproject.com/en/4.1/ref/forms/fields/#error-messages

Reply:
`versionadded:: 5.0` annotation is missing. I'd also document available keys and add release notes.

Reply:
I always forget the version tag 🤦 . Thank you!

This is how it looks like:

<img width="788" alt="image" src="https://user-images.githubusercontent.com/16822952/231030602-a39f352a-cd64-4cd8-afb1-411ffdc0699a.png">